doc/specs/#166 - Font Support.md

```diff
@@ -0,0 +1,416 @@
+- [IDWriteFontCollection::GetFontFamily](https://learn.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwritefontcollection-getfontfamily) - Creates the font family object using a provided index. Must be a valid index from the font collection.
+- [IDWriteFontList::GetFont](https://learn.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwritefontlist-getfont) - Gets the font from a family with a zero-based index
+- [IDWriteFont::GetInformationalStrings method](https://learn.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwritefont-getinformationalstrings) - Gets specific informational strings such as font full name, author, etc.
+- [IDWriteLocalFontFileLoader::GetFilePathFromKey](https://learn.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwritelocalfontfileloader-getfilepathfromkey) - Obtains the absolute font file path from a font file reference key.
+- [SHGetPropertyStoreFromParsingName](https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-shgetpropertystorefromparsingname) - Returns the property store for a given path to obtain details like the file's title.
+
+
+## Manifest Changes
+- Addition of `font` to `installerType` and `nestedInstallerType`
+
+Manifest Example:
+```yaml
+PackageIdentifier: Microsoft.CascadiaCode
+PackageName: Cascadia
+PackageVersion: 2404.23
+Installers:
+- InstallerType: zip
+  InstallerUrl: https://github.com/microsoft/cascadia-code/releases/download/v2404.23/CascadiaCode-2404.23.zip
+  InstallerSha256: a911410626c0e09d03fa3fdda827188fda96607df50fecc3c5fee5906e33251b
+  NestedInstallerType: font
+  NestedInstallerFiles:
+  - RelativeFilePath: ttf/CascadiaCodeItalic.ttf
+  - RelativeFilePath: ttf/CascadiaMonoNFItalic.ttf
+ManifestType: installer
+ManifestVersion: 1.9.0
+```
+
+## Font Index
+A new `PredefinedSource` will be created for `Fonts`. `PredefinedInstalledFontFactory` will be defined that creates a SQLite index from the installed font information. The table below shows how a font is analogous to a package.
+
+| Property | Package | Font |
+| -------- | ------- | ---- |
+| Id    |  MSIX\NotepadPlusPlus_1.0.0.0_neutral__7njy0v32s6xk6    |  Machine\Arial\Bold  |
+| Version    |  1.0.0.0  |  7.01  |
+| Name    |  NotepadPlusPlus   |  Arial Bold  |
+| PackageFamilyName    |  NotepadPlusPlus_1.0.0.0_neutral__7njy0v32s6xk6    |  Arial  |
+| InstalledLocation    |   N/A    | %LOCALAPPDATA%/Microsoft/Windows/Fonts/Arial.ttf |
+| InstalledScope    |       |   User |
+
+> The unique identifer for each font will be a combination of the scope, font family name, and font face name.
+
+> The version of the font only exists at the font face level and not at the family level
+
+## CLI Behavior / Implementation Details
+
+New Command: `winget fonts`
```

Comment:
Will there be any new GPO to prevent the installation of fonts, or uninstallation of specific fonts? 

Thinking about how an enterprise could use a custom standard font for all of their branding / marketing and would want to ensure that users at the company can't uninstall it using WinGet

Reply:
Also, how will this interact with existing Font GPOs, if at all?